test/karma.conf.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(config) {
 
   // On Travis CI, we can only run in Firefox.
```

Comment:
This comment is a bit out of date. Would be worth mentioning why firefox isn't included.

Reply:
Probably best to remove this comment.